### PR TITLE
chore: use SampleLoader for running sample tests

### PR DESCRIPTION
**Pull request by bajajneha27**

### CODE REVIEW

File: samples/authenticate_implicit_with_adc.rb [hunk 1]
```diff
@@ -40,7 +40,7 @@ def authenticate_implicit_with_adc project_id:
   buckets.each do |bucket|
     puts bucket.name
   end
-  puts "Listed all storage buckets."
+  puts "Plaintext: Listed all storage buckets."
```
Comment (dazuma) [inline samples/authenticate_implicit_with_adc.rb:43]: Why this change? (Out of curiosity)

**Review by dazuma — APPROVED** [review]

File: samples/authenticate_implicit_with_adc.rb (hunk 1 above)
Comment (bajajneha27) [reply samples/authenticate_implicit_with_adc.rb:43]: 😅  I actually can't remember now. I think I put it there while experimenting with sample_loader.
